provider/spark.go

```diff
@@ -2264,6 +2264,10 @@ func (spark *SparkOfflineStore) GetTrainingSet(id ResourceID) (TrainingSetIterat
 	return fileStoreGetTrainingSet(id, spark.Store, spark.Logger)
 }
 
+func (spark *SparkOfflineStore) GetTrainingSetTestSplit(id ResourceID, testSize float32, shuffle bool, randomState int) (TrainingSetIterator, TrainingSetIterator, func() error, error) {
+	return nil, nil, nil, nil
```

Comment:
should we add a `not implemented` or rather `not supported for Spark` error message?